### ISSUE TITLE
NPM

### ISSUE THREAD

**Issue by KK-swe** [darkroomengineering/lenis#50]
Hello. I tried to install it and build with npm but I cant get a demo html. Please can you tell me what I do wrong? Best regards, Karl

**Comment by arzafran** [comment]
Hello! can you please provide a codesandbox or codepen with how you're using it? there's not much we can do with the current information

**Comment by KK-swe** [comment]
Hello Arzafran. Thank you for the answer. I try to install it on my mac with npm. Then I used npm run build but there was no dist map with a html file as usual. I tried solve it a several days but I cannot find a solution about how to build it. Best wishes. p.s.: This code is fantastic and you are amazing, only me fail.

**Comment by clementroche** [comment]
Are you trying to build the website or the package ? What do you mean by "as usual" ? Can you list the commands you typed in your terminal because it's hard to understand what you're trying to do.

**Comment by KK-swe** [comment]
Hello Clement. With usual I mean, (I am not an expert) npm i and npm run build gives an editable html structure. I used pnpm i and the pnpm dev in terminal and was able to see the website in the browser but I do not know how I can export/build so I can get the editable html files. My background and goal; I am a veteran with task to save lives in war around the world. It means deliver food and medial supply and help children and woman to be evacuated. ( https://sverigesradio.se/artikel/karl-sov-hos-en-soldat-i-ukraina-i-natt-vet-inte-om-han-kommer-overleva      >this is from our national news) Now I will collect wisdom, experiences and photography and videos to illustrate the suffering and motivate people to peace. I created an Awwwards winning Wordpress site for it but that is not enough. I will give the collection more feeling, that is what Lentis means for me. I saw Lentis and I felt in love with it. I cannot write JS but I can manage/change it and HTML/CSS is not a problem. I am familiar also with glb. What I try to do, get a working demo, study it and build/design the site I see in my head. I should be happy if this idea can work. I see what Lentis is and it is simply amazing.
